src/config.c

```diff
@@ -91,6 +91,14 @@ configEnum aof_fsync_enum[] = {
     {NULL, 0}
 };
 
+configEnum memory_alloc_policy_enum[] = {
+    {"only-pmem", MEM_POLICY_ONLY_PMEM},
```

Comment:
Please follow the order defined in redis.conf, so:
first - only-dram,
second - only-pmem,
or modify redis.conf description.